stdlib/Markdown/src/Markdown.jl

```diff
@@ -9,7 +9,7 @@ literals `md"..."` and `doc"..."`.
 """
 module Markdown
 
-import Base: AnnotatedString, AnnotatedIOBuffer, show, ==, with_output_color, mapany
+import Base: AnnotatedString, AnnotatedIOBuffer, show, ==, with_output_color, mapany, unsafe_takestring!
```

Comment:
Why is this being imported here?

Reply:
Unnecessary, perhaps a leftover from an earlier change. Changed.